src/reactjs/useSyncExternalStore.native.ts

```diff
@@ -0,0 +1,2 @@
+// @ts-ignore
+export { useSyncExternalStore } from 'use-sync-external-store/shim/index.native.js'
```

Comment:
please have a look at the prettier errors in the pipeline. I think it comes from the missing newline in both files.

Reply:
I added the newline